### DISCUSSION TITLE
Firmware for RLC-510A with model IPC_MS1NA45MP?

### DISCUSSION THREAD

**Discussion by ryan-roberts** [AT0myks/reolink-fw-archive#39] [Missing firmware]
Just got a new RLC-510A on Amazon that according to the camera's setting Info screen came with hardware version IPC_MS1NA45MP. The firmware off of [reolink](https://reolink.com/download-center/) doesn't want to load, just gives me error "Update Failed Failed to recognize the file format. Please try again." which is a problem as it's being provided to it in its expected .pak format from the .zip.
![reolink_rlc-510A_systeminfo](https://github.com/AT0myks/reolink-fw-archive/assets/20831761/bbc9ed62-ccd2-4286-a86f-92981cec3e52)
![reolink_rlc-510A-photo](https://github.com/AT0myks/reolink-fw-archive/assets/20831761/1946f90a-b868-4961-b3c2-087b032e45ab)

**Comment by AT0myks** [comment]
This is a new hardware version for this model. The download center only has a firmware for hardware version IPC_523128M5MP which is not compatible with your device, and even if it was it's an older version than the one your camera has. Both the model name and hardware version must match when applying a firmware update. If you need a different firmware you can contact their support, otherwise you don't have anything to do. In the future Reolink might provide a firmware update for this specific hardware version, but for now it's possible you already have the latest version.
